app/assets/javascripts/read/src/reducers/map.js

```diff
@@ -40,6 +41,13 @@ const handlers = {
     choropleth: action.code,
   }),
 
+  [RESET_MAP_SETTINGS]: (state) => ({
+    baseLayerSlug: null,
```

Comment:
There is a constant `initialState` in this file with the same values. Can you use that here instead of defining them all anew?